src/main/java/gravitas/Gravitas.java

```diff
@@ -0,0 +1,52 @@
+package gravitas;
+
+import gravitas.command.Command;
+import gravitas.exception.DukeException;
+import gravitas.parser.Parser;
+import gravitas.storage.Storage;
+import gravitas.tasklist.TaskList;
+
+
+/**
+ * Represents the main class of the program.
+ */
+public class Gravitas {
+
+    private Storage storage;
+    private TaskList tasks;
```

Comment:
Correct use of variable tasks when naming variables that stores more than one task